contributors/devel/sig-testing/writing-good-conformance-tests.md

```diff
@@ -34,14 +34,16 @@ Instead, a better approach may be to duplicate the test, and make your
 modifications, thus preserving the original test as-is.
 
 Any existing e2e test should have gone through a standard review
-process by a respective [SIG](https://github.com/ii/k8s-community/tree/writing-good-conformance-tests) in order to
-have been merged in, which means that the test was most likely
+process by a respective [SIG](https://github.com/kubernetes/community/blob/master/sig-list.md)
+in order to have been merged in, which means that the test was most likely
 reviewed by domain-specific experts. This can be useful to note when
 modifying tests to better suit conformance concerns, or meet
-[requirements](#Requirements).
+[requirements](#Requirements). That said, existings tests are of varying levels
+of quality.  it is important to verify that the test actually does validate the
```

Comment:
It looks like there is an extra space after the period and "it" should be "It"

Reply:
thanks, fixed.